src/framework/components/sound/component.js

```diff
@@ -344,19 +344,19 @@ class SoundComponent extends Component {
      *
      * @param {string} name - The name of the slot.
      * @param {object} [options] - Settings for the slot.
-     * @param {number} [options.volume=1] - The playback volume, between 0 and 1.
-     * @param {number} [options.pitch=1] - The relative pitch, default of 1, plays at normal pitch.
-     * @param {boolean} [options.loop=false] - If true the sound will restart when it reaches the end.
-     * @param {number} [options.startTime=0] - The start time from which the sound will start playing.
-     * @param {number} [options.duration=null] - The duration of the sound that the slot will play
+     * @param {number} [options.volume] - The playback volume, between 0 and 1.
+     * @param {number} [options.pitch] - The relative pitch, default of 1, plays at normal pitch.
```

Comment:
do we want to add `Defaults to XXX` in all these instead?

Reply:
Done